tools/common.ps1

```diff
@@ -180,6 +180,27 @@ function Get-CoreNetCiArtifactPath {
     return "$RootDir\artifacts\corenet-ci-$Commit\vm-setup\$Name"
 }
 
+function Get-ArtifactBinPathBase {
+    param (
+        [Parameter()]
+        [string]$Config,
+        [Parameter()]
+        [string]$Arch
+    )
+
+    # Convert to Windows format
```

Comment:
i saw more conversions in the xdp.props file: 

```
    <WinPlat Condition="'$(Platform)' == 'Win32'">x86</WinPlat>
    <WinPlat Condition="'$(Platform)' == 'x64'">amd64</WinPlat>
    <WinPlat Condition="'$(Platform)' == 'ARM64'">arm64</WinPlat>
    <WinPlat Condition="'$(Platform)' == 'ARM64EC'">arm64ec</WinPlat>
```

should we transform all of those here, too?

Reply:
Since we don't support anything else in the project, I think it's fine as is for now.